include/fiction/algorithms/physical_design/post_layout_optimization.hpp

```diff
@@ -219,9 +231,6 @@ void add_fanout_to_route(const tile<Lyt>& fanout, bool is_first_fanout, fanin_fa
 template <typename Lyt>
 [[nodiscard]] fanin_fanout_data<Lyt> get_fanin_and_fanouts(const Lyt& lyt, const tile<Lyt>& op) noexcept
 {
-    static_assert(is_gate_level_layout_v<Lyt>, "Lyt is not a gate-level layout");
-    static_assert(is_cartesian_layout_v<Lyt>, "Lyt is not a Cartesian layout");
```

Comment:
Why were these `static_assert`s removed?

Reply:
I understand. And you're probably right. I would keep them anyways as they help to find and fix compiler errors during development